ext/spl/tests/heap_isempty_variation_001.phpt

```diff
@@ -2,14 +2,14 @@
 SPL: SplHeap: isEmpty argument variation.
 --FILE--
 <?php
-class SplHeap2 extends SplHeap{
 
-  public function compare() {
-           return -parent::compare();
-       }
+class SplHeap2 extends SplHeap {
+  public function compare($a, $b) {
+    return -parent::compare($a, $b);
```

Comment:
_(Stupid question)_: what does `-` do here?

Reply:
The `-` prefix negates the return value, so if the right operand returns 8, then this will return -8